core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -614,7 +614,7 @@ HIVE_TAR=${HIVE_PREFIX}.tar.gz
 HBASE_MIRROR_URL=http://archive.cloudera.com/cdh5/cdh/5
 HBASE_TAR=hbase-${HBASE_DEP_VER_CDH}.tar.gz
 if [[ "$HBASE_DISTRO" = "HDP" ]]; then
-    HBASE_TAR=hbase-${HBASE_DEP_VER_HDP}.tar.gz
+    HBASE_TAR=hbase-1.1.2.2.3.2.0-2950.tar.gz
```

Comment:
Prashanth brought this up already. I suggested that It would be better to create a soft link is created at the local repository to point to the HBASE_DEP_VER_HDP rather than changing the install_local_hadoop.